eth-contracts/package.json

```diff
@@ -3,19 +3,18 @@
   "version": "0.0.1",
   "description": "CKB SPV proofs in Solidity",
   "scripts": {
-    "compile": "npx buidler compile",
-    "test": "npx buidler test",
+    "compile": "npx hardhat compile",
```

Comment:
The `npx` in package.json can be  omitted. When scripts in package.json executed, it will load local env as we want.